docs/cni/rkt/README.md

```diff
@@ -140,8 +137,7 @@ With the networks created, let's start a container on `calico-01`
 ### On calico-01
 Create a webserver using an `nginx` container. We'll put this on `net1` network.
 
-Because `rkt` doesn't support daemonizing containers directly, we'll use [systemd-run][systemd-run]
-to create a transient systemd service.
+We'll use systemd run to create the webserver service.
 
     sudo systemd-run rkt run --store-only=true --insecure-options=image --net=net1 --mount volume=myvol,target=/var/run --volume=myvol,kind=empty docker://nginx
  
```

Comment:
Two things RE the next section:

1. My nginx node took a few seconds to come up, presumably while it was pulling the rkt container.might want to warn the reader, since I immediately assumed something was wrong.
2. calico-node should show up in the output when they run rkt list ( as it showed up in mine). I think we might as well include it in the sample output.